hail_search/queries/mito.py

```diff
@@ -112,6 +118,215 @@ class MitoHailTableQuery(BaseHailTableQuery):
     }
     SORTS[PATHOGENICTY_HGMD_SORT_KEY] = SORTS[PATHOGENICTY_SORT_KEY]
 
+    def _import_and_filter_entries_ht(
+        self, project_guid: str, num_families: int, project_sample_type_data, **kwargs
+    ) -> tuple[hl.Table, hl.Table]:
+        sample_types = set(project_sample_type_data.keys())
+        if len(sample_types) == 1:
+            return super()._import_and_filter_entries_ht(project_guid, num_families, project_sample_type_data, **kwargs)
+
+        self._has_both_sample_types = True
```

Comment:
I don't think so, the behavior for a variant that's present in one sample type only for a family should be the same as it is in searches today, even though it will follow the 'both sample types' flow instead of the original flow - [this line](https://github.com/broadinstitute/seqr/pull/4330/files#diff-71932dea9e59d8d1ce70d88444658acffce2f3865c9afb11b57f93daf6610f66R304) will treat the missing sample type entries as failing checks for that sample type and only keep the variant if it passes in the one sample type.

Reply:
This is also covered in the first test case in `test_both_sample_types_search` - the search has multiple projects and only one project has samples from both sample types.